app/main/views/reset_password.py

```diff
@@ -28,8 +28,8 @@
     """
 )
 
-PASSWORD_UPDATED_MESSAGE = "You have successfully changed your password."
-PASSWORD_NOT_UPDATED_MESSAGE = "Could not update password due to an error."
+PASSWORD_UPDATED_MESSAGE = "Your password has been successfully changed."
```

Comment:
```suggestion
PASSWORD_UPDATED_MESSAGE = "Your password has been changed successfully."
```

I can never remember what's right (_something something split infinitive something_?), but this sounds more correct to me.

Googling the suggested phrase brings up 18 million+ results vs the existing phrase with 250K.

Reply:
Interesting! It felt more natural to me as well.